src/controllers/automation/automationGridCtrl.js

```diff
@@ -22,6 +22,7 @@
     $scope.rows = 10;
     $scope.tasksQuantity = '';
     $scope.totalTasks = 0;
+    $scope.automationTemplateSelected = -1;
```

Comment:
Very minor, but I think that it would be better to call it `idAutomationTemplateSelected`, or something that gives us the idea it is an id. What do you think?